pyanaconda/ui/gui/spokes/advanced_storage.py

```diff
@@ -774,6 +789,20 @@ def on_add_dasd_clicked(self, widget, *args):
         # storage are displayed in the UI.
         self.refresh()
 
+    def on_reconfigure_nvdimm_clicked(self, widget, *args):
+
+        namespaces = self.pages[3].get_selected_namespaces()
+
+        dialog = NVDIMMDialog(self.data, self.storage, namespaces)
```

Comment:
Is here a reason to have those two blank lines above? I think you can remove them.